### PR TITLE
Fix unit test rounding

### PR DESCRIPTION
**Pull request by aavenel**
Test was Ok but failed due to some floating point errors.

### CODE REVIEW

**Comment by JocelynDelalande** [comment]
@aavenel instead of fixing the test, would using `from __future__ import division` solve the issue ?

(it's a bit of a wild guess…)

File: budget/tests.py [hunk 1]
```diff
@@ -498,7 +498,8 @@ def test_rounding(self):
         result[models.Project.query.get("raclette").members[0].id] = 8.12
         result[models.Project.query.get("raclette").members[1].id] = 0.0
         result[models.Project.query.get("raclette").members[2].id] = -8.12
-        self.assertDictEqual(balance, result)
+        for key, value in balance.iteritems():
```
Comment (JocelynDelalande) [inline budget/tests.py]: Maybe just add a comment to explain why you have to do that weird (but required) thing.

**Comment by JocelynDelalande** [comment]
> @aavenel instead of fixing the test, would using from __future__ import division solve the issue ?
> 
> (it's a bit of a wild guess…)

Auto-answering me : no (just tested), so your way seem the right one :+1:

**Comment by aavenel** [comment]
@JocelynDelalande The root cause is that we're using floating point to store currency instead of fixed point (Decimal in python). Replacing all the fields would probably be a big change so we're better off with a few rounding for now.

**Comment by JocelynDelalande** [comment]
> @JocelynDelalande The root cause is that we're using floating point to store currency instead of fixed point (Decimal in python). Replacing all the fields would probably be a big change so we're better off with a few rounding for now.

The comment was not for me but to put above your code in source.

**Comment by almet** [comment]
Hi, this is a friendly reminder: Alexandre, could you add the comment to the code in another commit? We would then be good to merge!

**Comment by aavenel** [comment]
Added comment and rebased ! :)
All tests should be green !

**Comment by JocelynDelalande** [comment]
All right, thanks @aavenel ; it's good to be green :-)

**Pull request merged**